docs/_jul_milestone_.md

```diff
@@ -0,0 +1,23 @@
+# Monthly Milestone - July 2020
+
+## Overview
+
+- Docs issue takedown!
+
+## Goals
+
+- [ ] Docs improvements
+
+  _Please add your suggestions here. Also see
+  https://github.com/strongloop/loopback-next/issues?q=is%3Aissue+is%3Aopen+label%3ADocs+label%3A2020Q3_
+
+- [ ] Feature parity
+
+  - [ ] Execute raw NoSQL queries (e.g. MongoDB) #3342
+
+- [ ] Fix GVT issues
+  - [ ] [GSSC-GVT]English String when input command apic lb4 app -h #5654
+  - [ ] [GSSC-GVT]English String when input command apic lb4 repository #5656
+  - [ ] [GSSC-GVT]English String when input command apic lb4 model #5657
+
```

Comment:
Chores:

- update dependency mem-fs-editor to v7 #5624

Reply:
Tooling improvements:

- [api-extractor] Incorrect handling of interfaces enhancing a class with the same name https://github.com/microsoft/rushstack/issues/1921